src/99-appendices/09-entities.md

```diff
@@ -17,15 +17,15 @@ each is given in the [Entity Table](04-entity-table.md).
 
 Full name: Subject
 
-Format: `sub-<label>`
+Format: `subject-<label>`
```

Comment:
@tsalo any idea why this is happening?

Reply:
Argh, there's a bug in the generation code! The fix is buried in #324. I'm not sure if I should show you how to fix it here or open a separate PR with the fix...

Reply:
For simplicity's sake, I'm going to ask you to make the change, if you're willing. At the following line, please replace `entity` with `entity_info['entity']`:

https://github.com/bids-standard/bids-specification/blob/580183fe1af767b4ea7ee3b0e9459713c0c6b658/tools/bids_schema.py#L227

Reply:
@tsalo now it looks ok 👍